public_suffix_list.dat

```diff
@@ -13924,6 +13924,16 @@ pixolino.com
 // Submitted by Vasiliy Sheredeko <piphon@gmail.com>
 na4u.ru
 
+// IONOS SE : https://www.ionos.com/,
+// IONOS Group SE: https://www.ionos-group.com/
+// submitted by Henrik Willert <security@ionos.com>
+
```

Comment:
Please remove the empty line (to avoid breaking some parsers)

Reply:
Fixed in https://github.com/publicsuffix/list/pull/2083/commits/ff20b6bd1aec6901d93a1ed9c574db0d681adf4e :heavy_check_mark: